indra/sources/sofia/api.py

```diff
@@ -27,7 +28,10 @@ def process_table(fname):
         rel_sheet = book['Causal']
     event_sheet = book['Events']
     entities_sheet = book['Entities']
-    sp = SofiaProcessor(rel_sheet.rows, event_sheet.rows, entities_sheet.rows)
+    sp = SofiaProcessor(input_object=(rel_sheet.rows, event_sheet.rows,
+                                      entities_sheet.rows))
+    if not sp.statements:
+        return None
```

Comment:
To match the standard behavior of other processors, we shouldn't do this return None here. sp.statements should just be an empty list in case no statements were extracted.

A _different_ scenario in which case some serious issue happened while processing the input  (e.g. the input file doesn't exist or the web service call failed) it may be appropriate to return a None processor.

Reply:
Fixed it in latest commit.